fastai/vision/learner.py

```diff
@@ -37,7 +37,7 @@ def _alexnet_split(m:nn.Module): return (m[0][0][6],m[1])
 
     models.densenet121:{**_densenet_meta}, models.densenet169:{**_densenet_meta},
     models.densenet201:{**_densenet_meta}, models.densenet161:{**_densenet_meta},
-    models.vgg11_bn:{**_vgg_meta}, models.vgg13_bn:{**_vgg_meta}, models.vgg16_bn:{**_vgg_meta}, models.vgg19_bn:{**_vgg_meta},
+    models.vgg16_bn:{**_vgg_meta}, models.vgg19_bn:{**_vgg_meta},
```

Comment:
Be careful, you are removing functionality here that's not linked to your PR.

Reply:
Sure, will fix